bin/pycbc_compress_bank

```diff
@@ -147,17 +151,21 @@ for ii in range(imin, imax):
     bank.delta_f = df
     bank.N = N
     bank.filter_length = N/2 + 1
-    bank.f_lower = fmin
-    bank.kmin = int(bank.f_lower / df)
     # scratch space
     decomp_scratch = FrequencySeries(numpy.zeros(N, dtype=dtype), delta_f=df)
     # generate the waveform
     htilde = bank[ii-imin]
-    if numpy.abs(htilde[bank.kmin]) == 0:
+    tmplt = bank.table[ii-imin]
+    if args.low_frequency_cutoff:
+        fmin=args.low_frequency_cutoff
+        bank.f_lower = fmin
+    else:
+        fmin=tmplt.f_lower
```

Comment:
I was just keeping it consistent with how you redefined `bank.f_lower` in the loop with the `--low-frequency-cutoff` in the current (master) version of the code. But yes, this if-else is not required. I'll remove it.